release_creation/snapshot/download.sh

```diff
@@ -0,0 +1,6 @@
+python -m pip install --upgrade pip
```

Comment:
I'd suggest strongly throwing pipefail and errexit flags on this script just as good scripting practice.

Reply:
added errexit not sure pipefail would help here but can add too!